src/ss_ros.py

```diff
@@ -38,8 +38,8 @@ class ss_ros():
     # set up rostopics we publish: commands to the game (on a tablet or on a
     # PC/touchscreen), and commands to the robot
     game_pub = rospy.Publisher('opal_tablet_command', OpalCommand,
-            queue_size = 10)
-    robot_pub = rospy.Publisher('robot_command', RobotCommand, queue_size = 10)
+            queue_size = 10) #cmhuang: change the topic to /sar/opal_tablet_command
```

Comment:
Please put the comment on its own line above the code it refers to.

Did you update SAR Opal base to use the new topic name?